<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Confirmation</class>
 <widget class="QMainWindow" name="Confirmation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>313</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0.435644, y1:0.767, x2:1, y2:0, stop:0 rgba(20, 19, 5, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Top_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CONFIRM INFORMATION?</string>
    </property>
   </widget>
   <widget class="QLabel" name="TopBelow_Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Please double check all information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Confirm_Button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIRM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GoBack_Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GO BACK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
